src/ducks/settings/TogglePane.jsx

```diff
@@ -82,9 +81,4 @@ TogglePane.propTypes = {
   saveDocument: PropTypes.func.isRequired
 }
 
-export default compose(
-  translate(),
-  // We keep `connect` just so the component still receive the `dispatch` function as prop
-  // This has to be removed when we handle the mutations with the new cozy-client
-  connect()
-)(TogglePane)
+export default compose(translate())(TogglePane)
```

Comment:
Il y a plus besoin de compose du coup.

```suggestion
export default translate()(TogglePane)
```